### DISCUSSION TITLE
Edit Column Form Doesn't Show But Blocks Application

### DISCUSSION THREAD

**Discussion by MustacheMushroom** [pawelsalawa/sqlitestudio#5173] [General]
### Details
When trying to edit a column's details, either by double clicking the column or by selecting it, then clicking the Edit Column button when in the Structure tab, the application behaves like the form has opened, but I don't see it on the screen.

### Steps to reproduce
1. Open an SQLite db file.
2. Open a table.
3. Attempt to open a column as described above.
4. Return to main application by pressing escape which seems to close the invisible window.
5. Cry. (I've tried closing and reopening the application, but to no avail, rebooting seems to fix it)

### Operating system
Windows 11 Enterprise 23H2

### SQLiteStudio version
3.4.9 (but has happened with the last 2 versions as well) installer version (but also happened with portable version).

**Comment by pawelsalawa** [comment]
Your dialog opens at coordinates 0x0 (top left corner of your desktop). Do you happen to have your desktop on multiple monitors and currently one of them is turned of?

Try with 3.4.10, it has some relevant fix for it, but it may not work immediately. If you already open a dialog at 0x0, then 3.4.10 won't help until you move the window to the correct position at least once. From that moment it will always open at that position.

**Reply by mathtester** [reply]
It looks like the same issue as #5161

**Comment by mathtester** [comment]
Issue #5161, where the same problem described here was observed, has been resolved. That issue is for portable version. If someone can check an installed version, this bug can also be closed.
